pegasus/sites.v3/code.org/public/ai/index.haml

```diff
@@ -130,7 +130,18 @@ theme: responsive_full_width
           %a.link-button{href: CDO.studio_url("/courses/self-paced-pl-aiml-2023")}
             =hoc_s(:call_to_action_self_paced_pl)
 
-= view :section_divider_line
+
+%section.bg-neutral-light
+  .wrapper{style: "display: flex; justify-content: space-between; align-items: center; flex-wrap: wrap"}
```

Comment:
You can use classes here instead of inline styles: https://github.com/code-dot-org/code-dot-org/blob/f4e18c9ae95ca76d3b235208322494adf1bfc0cf/pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss#L81-L98